src/pages/home/report/ReportFooter.js

```diff
@@ -68,7 +68,7 @@ function ReportFooter(props) {
     return (
         <>
             {hideComposer && (
-                <View style={[styles.chatFooter, props.isSmallScreenWidth ? styles.mb5 : null]}>
+                <View style={[styles.chatFooter, isArchivedRoom || isAnonymousUser ? styles.pt5 : {}, props.isSmallScreenWidth ? styles.mb5 : null]}>
```

Comment:
To be consistent with composer padding, I think 16px makes more sense. And what's the advantage of pt4 vs mt4?

Reply:
@mkhutornyi 16px looks better to me too, updated. Seems there is not any difference between margin and padding, updated with mt4 to keep same style for top and bottom